### PR TITLE
Initial template setup

### PR DESCRIPTION
**Pull request by waqaskanju**
-Read me file changed.
- Node added in ignore
- New Description branch added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,58 +5,7 @@ This Repo is for using best practices in Mircoverse
...
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+# Author
```
Comment (MeqdamAlqudah) [inline README.md:10]: - [ ] Please README file should be descriptive? AKA:[Microverse README](https://github.com/microverseinc/readme-template) template should be used.

File: .gitignore [hunk 2]
```diff
@@ -2,4 +2,8 @@
 .github
```
Comment (MeqdamAlqudah) [inline .gitignore:2]: - [ ] Please your commit messages should describe an action and they should be in the present tense like `Add HTML ` `Change CSS` and so on
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @waqaskanju , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (waqaskanju) [reply README.md]: Added Back. Thanks.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**